2019.08 release notes (2019-10-01)

```diff
@@ -0,0 +1,41 @@
+# 2019.08 release notes (2019-10-01)
```

Comment:
if you rename the file to have a `.md` extension, you can "view file" on github and see the rendered markdown to make sure there are no problems with the markdown.

Reply:
Great! Sorry I forgot to add that!